toucan_connectors/rok/rok_connector.py

```diff
@@ -1,12 +1,12 @@
 import base64
 import json
 from datetime import datetime, timedelta
-from json import JSONDecodeError
 
 import pandas as pd
 import requests
 from jwt import encode
 from pydantic import Field
+from simplejson import JSONDecodeError
```

Comment:
Do you know why we need to import that from the simplejson module ?
Is that related to the jwt module using it internally?

Reply:
requests seems to be raising simplejson.errors.JSONDecodeError when failling to parse a response object with .json() :

```
self = <simplejson.decoder.JSONDecoder object at 0x7f9fed2bcbb0>, s = '<Fault xmlns= ...', idx = 0, _w = <built-in method match of re.Pattern object at 0x7f9fed9391b0>, _PY3 = True

    def raw_decode(self, s, idx=0, _w=WHITESPACE.match, _PY3=PY3):
        """Decode a JSON document from ``s`` (a ``str`` or ``unicode``
        beginning with a JSON document) and return a 2-tuple of the Python
        representation and the index in ``s`` where the document ended.
        Optionally, ``idx`` can be used to specify an offset in ``s`` where
        the JSON document begins.
    
        This can be used to decode a JSON document from a string that may
        have extraneous data at the end.
    
        """
        if idx < 0:
            # Ensure that raw_decode bails on negative indexes, the regex
            # would otherwise mask this behavior. #98
            raise JSONDecodeError('Expecting value', s, idx)
        if _PY3 and not isinstance(s, str):
            raise TypeError("Input string must be text, not bytes")
        # strip UTF-8 bom
        if len(s) > idx:
            ord0 = ord(s[idx])
            if ord0 == 0xfeff:
                idx += 1
            elif ord0 == 0xef and s[idx:idx + 3] == '\xef\xbb\xbf':
                idx += 3
>       return self.scan_once(s, idx=_w(s, idx).end())
E       simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

Reply:
Thanks for clarifying it!

Makes me think that I think the ROK connector depends on the `pyjwt` module, and we should add it here: https://github.com/ToucanToco/toucan-connectors/pull/222/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R31)
I'm surprised we have to indicate simplejson as dependecy here, as it's probably already installed by the package that raises the error. Maybe we can remove it ?

Reply:
I changed that, let's see if the CI is ok

Reply:
![image](https://user-images.githubusercontent.com/71276735/96229435-3181ca80-0f97-11eb-8b7d-fb222882950f.png)

Got that when removing simplejson from required dependicies :(

Reply:
simplejson finally added to dependencies.